src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersAxis.tsx

```diff
@@ -103,17 +119,29 @@ export default function TableHeadersAxis({
               <div className={styles.groupsContainer}>
                 {values.map((group: Filter[], index: number) => {
                   const key = `group-${index}`;
+                  const groupName = `${name}[${index}]`;
+                  const groupLegend = getGroupLegend(group);
+                  const groupId = getTableHeaderGroupId(groupLegend);
```

Comment:
Purely a stylistic suggestion, but I wonder if declaring an interface with these four properties might be clearer than instantiating four variables? 

Then, since `id`, `index`, and the `moveUp`/`moveDown` methods are all passed down twice, we could even have each of them extend it in their props then just use the spread operator all the way down. 

Doesn't make a lick of difference, just thought it might be more concise :)